Localization/output/autodrive.ipynb

```diff
@@ -0,0 +1,128 @@
+{
```

Comment:
Let's not add python notebooks to github, they are a pain in version control systems. Please add
the line *.ipynb to .gitignore file and remove these files from pull request.